invokeai/app/invocations/compel.py

```diff
@@ -66,25 +64,17 @@ class CompelInvocation(BaseInvocation):
 
     @torch.no_grad()
     def invoke(self, context: InvocationContext) -> ConditioningOutput:
-        tokenizer_info = context.services.model_manager.get_model(
-            **self.clip.tokenizer.model_dump(),
-            context=context,
-        )
-        text_encoder_info = context.services.model_manager.get_model(
-            **self.clip.text_encoder.model_dump(),
-            context=context,
-        )
+        tokenizer_info = context.models.load(**self.clip.tokenizer.model_dump())
+        text_encoder_info = context.models.load(**self.clip.text_encoder.model_dump())
 
         def _lora_loader():
             for lora in self.clip.loras:
-                lora_info = context.services.model_manager.get_model(
-                    **lora.model_dump(exclude={"weight"}), context=context
-                )
+                lora_info = context.models.load(**lora.model_dump(exclude={"weight"}))
                 yield (lora_info.context.model, lora.weight)
                 del lora_info
             return
 
-        # loras = [(context.services.model_manager.get_model(**lora.dict(exclude={"weight"})).context.model, lora.weight) for lora in self.clip.loras]
+        # loras = [(context.models.get(**lora.dict(exclude={"weight"})).context.model, lora.weight) for lora in self.clip.loras]
```

Comment:
seems like this could just be removed, but I respect even updating the comments

Reply:
Heh, I think that was a find/replace